README.md

```diff
@@ -1,104 +1,46 @@
 DAGMC-viz Documentation
 =======================
 
-Note: Python 2.7 and visit are required.
-Python path needs to be set for visit.
-Default operators rely on Pseudocolor for spatial definition.
-mbconvert and mbgsets are used for bash options.
+The Direct Accelerated Geometry Monte Carlo visualization toolkit requires Python 2.7, VisIt, and MOAB, as well as the inclusion of VisIt and MOAB in the Python path. A user can input a geometry file and a data file and produce default interactive output in VisIt, with the option to save images of the plot windows as well. Functionality allowing the user to have full control of determining the output will be implemented in the future.
 
 ----------------------------------------
 
-1. MakeImagesTerminal.py can be used to load data.
-	
-		python MakeImagesTerminal.py -pl [plot list] -op [operator list]
+1. DataLoading.py can be used to create a default output in VisIt from an h5m geometry file and an h5m or vtk data file. The following four interactive plot windows are displayed in a 2x2 grid.
 
-    The terminal will prompt the user to insert lists of plots. Operators may also be added. Each plot and operator has unique settings that can be defined. Look at each plot and operator python file for specifics.
+      1. A cube with the default view normal to a corner.
+      2. XY plane slice through the center.
+      3. XZ plane slice through the center.
+      4. ZY plane slice through the center.
+      
+Each window has a mesh plot with the "STL_mesh" variable, a Pseudocolor plot with the "TALLY_TAG" variable, and the second, third, and fourth windows have Contour plots with the "ERROR_TAG" variable. 
 
-    To plot, the following format is used:
+```
+python DataLoading.py [geometry_file] [data_file] 
+```
+The user may indicate whether or not they would like each window to be saved as a png file in their current directory.
```

Comment:
... by adding the `-i` option to the command. 

Best to be explicit about it so readers don't have to parse the difference in the commands to know.